borgmatic/commands/arguments.py

```diff
@@ -865,7 +969,19 @@ def make_parsers():
     )
     borg_group.add_argument('-h', '--help', action='help', help='Show this help message and exit')
 
-    return top_level_parser, subparsers
+    merged_subparsers = argparse._SubParsersAction(
+        None, None, metavar=None, dest='merged', parser_class=None
+    )
+
+    for name, subparser in subparsers.choices.items():
+        merged_subparsers._name_parser_map[name] = subparser
+        subparser._name_parser_map = merged_subparsers._name_parser_map
+
+    for name, subparser in config_subparsers.choices.items():
+        merged_subparsers._name_parser_map[name] = subparser
+        subparser._name_parser_map = merged_subparsers._name_parser_map
+
+    return top_level_parser, merged_subparsers
```

Comment:
Any test for these two loops portion of the function? I'm still not 100% sure of what they do. :smile:

Reply:
They merge all the subsubparsers and our subparser into one. Here's the discussion - https://github.com/borgmatic-collective/borgmatic/pull/71#discussion_r1212166616

Reply:
Oh, I can't really say I do either, I was just trying things out and this ended up working.

Reply:
Now that I look at it again, this line in both loops confuses me too:

```python
        subparser._name_parser_map = merged_subparsers._name_parser_map
```